apps/deploy-web/src/utils/constants.ts

```diff
@@ -127,7 +127,7 @@ function getProviderProxyWsUrl() {
 export let selectedNetworkId = "";
 
 // 0.5AKT aka 500000uakt
-export const defaultInitialDeposit = 500000;
+export const defaultInitialDeposit = 5000000;
```

Comment:
It's supposed to be 0.5 AKT, any reason for the change?

Reply:
wasn't working for me this way. let's take a look together into this

Reply:
Hmm, maybe the sandbox doesn't have the same params. In any case, we have a `useDepositParams` hook that fetchs the deposit min from the on-chain params, so this variable is only used as the default value and shouldn't matter.

Reply:
ok, let's take a look at this. doesn't work for me 

``` ERROR (799): Error: Query failed with (6): rpc error: code = Unknown desc = failed to execute message; message index: 1: Deposit too low - 500000 < 5000000uakt: Deposit invalid [akash-network/akash-api@v0.0.67/go/node/deployment/v1beta3/params.go:54] With gas wanted: '0' and gas used: '91641' : unknown request```

Reply:
Perhaps the sandbox param hasn't been updated. We can probably remove this variable and use the `useDepositParams` everywhere we need this value to fix.

Reply:
ok, thanks for the hint, [this works](https://github.com/akash-network/console/pull/265/commits/8ed3422ec6a1c417c9f01b2249565ff3f46a5d18)